### ISSUE TITLE
rumdl-disable MD041 on first line of file does not work in MDX-style comment

### ISSUE THREAD

**Issue by polarweasel** [rvben/rumdl#538]
### rumdl version

0.1.44

### Installation method

Homebrew

### Bug description

As far as I can tell, rumdl responds to directives in the MDX comment style (for example, `{/* <!-- rumdl-disable --> */}`) for **every rule except MD041**.

For example, I can disable rumdl on the first line of a file with no H1 at the top, add a bare URL in the body, and rumdl flags only the MD041 violation for the missing heading, not the MD034 for the bare URL.

### Steps to reproduce

Rumdl (as expected) throws an MD041 and MD034 violation for this file:

```mdx
<Note>
This documentation is linted with http://rumdl.dev/
</Note>
```

```sh
$ rumdl check docs/snippets
docs/snippets/note.mdx:1:1: [MD041] First line in file should be a level 1 heading
docs/snippets/note.mdx:2:182: [MD034] URL without angle brackets or link formatting: 'http://rumdl.dev/' [*]
```

Inserting an HTML comment directive as the first line fixes the problem:

```markdown
<!-- rumdl-disable MD041 MD034 -->
<Note>
This documentation is linted with http://rumdl.dev/
</Note>
```

```sh
$ rumdl check docs/snippets

Success: No issues found in 1 file (18ms)
```

BUT, using an MDX-based SSG (Fern), I need to enclose HTML comments like this, and rumdl throws (only) an MD041 violation:

```mdx
{/* <!-- rumdl-disable MD041 MD034 --> */}
<Note>
This documentation is linted with http://rumdl.dev/
</Note>
```

```sh
$ rumdl check docs/snippets
docs/snippets/note.mdx:1:1: [MD041] First line in file should be a level 1 heading
```

### Expected behavior

All rules should recognize the directive within the MDX-style comment.

### Actual behavior

MD041 ignores the directive, but other rules do recognize it.

### Configuration

```toml
# most config omitted for brevity...

[global]
flavor = "mdx"
```

### Operating system

macOS

**Comment by rvben** [comment]
Fixed in v0.1.57. MD041's first-content-line detection was skipping HTML comments but not MDX comments (`{/* ... */}`), so the MDX comment line was treated as the first content line where the persistent disable hadn't taken effect yet.

**Comment by polarweasel** [comment]
Thank you! That was ridiculously quick turnaround. 🤩
